src/states/node.rs

```diff
@@ -848,6 +945,17 @@ impl Node {
         }
 
         self.add_to_routing_table(public_id, peer_id);
+
+        if let Some(prefix) = self.peer_mgr.routing_table().find_group_prefix(public_id.name()) {
+            let _ = self.send_section_list_signature(prefix, None);
```

Comment:
Do we need to send it our signatures directly? Won't it get them anyway from members of its own section (5 lines below)?